dm/pkg/ha/bound.go

```diff
@@ -202,46 +202,86 @@ func GetLastSourceBounds(cli *clientv3.Client) (map[string]SourceBound, int64, e
 // if source bound is empty, will return an empty sourceBound and an empty source config
 // if source bound is not empty but sourceConfig is empty, will return an error
 // if the source bound is different for over retryNum times, will return an error.
-func GetSourceBoundConfig(cli *clientv3.Client, worker, source string) (SourceBound, *config.SourceConfig, int64, error) {
+func GetSourceBoundConfig(cli *clientv3.Client, worker, source string) (map[string]SourceBound, map[string]*config.SourceConfig, int64, error) {
```

Comment:
can the worker parameter be empty?

Reply:
`worker` parameter can't be empty, it still can't get bounds or configs in all worker, it will return empty if `worker` parameter is empty.